test/jasmine/tests/treemap_test.js

```diff
@@ -1395,8 +1395,8 @@ describe('Test treemap tweening:', function() {
             _assert('update b to new position', 'd', 'b',
                 'M221.75,136L611,136L611,361L221.75,361Z'
             );
-            _assert('move B text to new position', 'transform', 'B', [221.75126, 0]);
-            _assert('move b text to new position', 'transform', 'b', [224.75150, 0]);
+            _assert('move B text to new position', 'transform', 'B', [222.75, 126]);
+            _assert('move b text to new position', 'transform', 'b', [225.75, 150]);
```

Comment:
Hmm. I'm getting these new jasmine test failures locally:

![image](https://user-images.githubusercontent.com/6675409/70746652-311fff80-1cf4-11ea-8ef8-6bbf200a3dc8.png)

Is that due to some local vs CI discrepancy?

Reply:
Strange. They pass on my machine as well as CI.

Reply:
Now fixed!!